helm/esignet/install.sh

```diff
@@ -17,9 +19,23 @@ function installing_esignet() {
 
   ./keycloak-init.sh
 
+  echo Please enter the recaptcha admin site key for domain $ESIGNET_HOST
+  read ESITE_KEY
+  echo Please enter the recaptcha admin secret key for domain $ESIGNET_HOST
+  read ESECRET_KEY
+
+  echo Setting up captcha secrets
+  kubectl -n $NS create secret generic esignet-captcha --from-literal=esignet-captcha-site-key=$ESITE_KEY --from-literal=esignet-captcha-secret-key=$ESECRET_KEY --dry-run=client -o yaml | kubectl apply -f -
+
   echo Copy configmaps
   ./copy_cm.sh
 
+  echo copy secrets
+  ./copy_secrets.sh
+
```

Comment:
Add esignet captcha env variable for config-server deployment yaml via kubectl command

Reply:
also shall we not add it to same mosip-captcha ?

Reply:
esignet is independed module that's why @syedsalman3753 told to create new name as 'esignet-captcha' saperately.